Added a text field.

jjunaid12 opened the pull request:
Added a text field for students to enter their consultation reason. Modified the UI design of the dialog to accommodate the new input field.

29-29 commented on src/dialogs/booking/BookAppointmentDialog.vue:
```diff
@@ -1,25 +1,37 @@
+      <q-card v-if="booking && isBookingAppear">
+        <q-form @submit="onSubmit" class="q-px-md q-py-sm">
+          <q-card-section class="q-mt-md">
+            <div class="text-subtitle1 text-primary">Reason for Booking</div>
```
Purpose?
```suggestion
            <div class="text-subtitle1 text-primary">Purpose</div>
```

29-29 committed:
fix: fixed default date value in the date picker

- changed the date button from dense to flat
- added a default date value in the date picker

29-29 committed:
Update src/dialogs/booking/BookAppointmentDialog.vue

jjunaid12 committed:
feat: add input text for consultation reason
